src/home/Footer.jsx

```diff
@@ -0,0 +1,118 @@
+import React from "react";
+import "./Footer.css";
+import { Link } from "react-router-dom";
+import github from "./../assets/images/github.svg";
+import linkdin from "./../assets/images/LinkedIn.svg";
+import twitter from "./../assets/images/Twitter.svg";
+import facebook from "./../assets/images/facebook.svg";
+export default function Footer() {
+  return (
+    <section id="footer">
+      <div className="footer-container">
+        <div className="main-flexbox">
+          <div className="flexbox-item">
+            <img className="logo" src="#" alt="Logo" />
+            <h4 className="text">
+              We are known for connecting top talents with top companies. We
+              make great possibilities happen.
+            </h4>
+          </div>
+          <div className="flexbox-item">
+            <h3 className="heading">Explore</h3>
+            <ul className="list">
+              <Link to="/" style={{ textDecoration: "none" }}>
+                <li className="li">Home</li>
+              </Link>
+              <Link style={{ textDecoration: "none" }}>
+                <li className="li">About Us</li>
+              </Link>
+              <Link style={{ textDecoration: "none" }}>
+                <li className="li">Blogs</li>
+              </Link>
+              <Link style={{ textDecoration: "none" }}>
+                <li className="li">FAQ</li>
+              </Link>
+              <Link style={{ textDecoration: "none" }}>
```

Comment:
You can add this style to a class instead of repeating this inline style for every element.

Reply:
Ya sure @Rahulm2310  👍